components/content_settings/core/browser/brave_content_settings_pref_provider_unittest.cc

```diff
@@ -64,10 +64,17 @@ void InitializeAllShieldSettingsInDictionary(
       kSessionModelPath,
       static_cast<int>(content_settings::SessionModel::Durable));
 
-  std::unique_ptr<prefs::DictionaryValueUpdate> per_resource_dict =
-      dictionary->SetDictionaryWithoutPathExpansion(
-          kPerResourcePath, std::make_unique<base::DictionaryValue>());
+  return dictionary->SetDictionaryWithoutPathExpansion(
+      kPerResourcePath, std::make_unique<base::DictionaryValue>());
+}
 
+void InitializeAllShieldSettingsInDictionary(
+    prefs::DictionaryValueUpdate* dictionary,
+    const base::Time& last_modified_time,
+    const int& value) {
```

Comment:
const int& -> int

Reply:
Oops! It's actually in many places in this file, fixed all of them.